src/java/com/sarality/app/view/dialog/BaseDialogFragment.java

```diff
@@ -27,8 +17,8 @@
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState) {
     AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-    getActionManager().setup(builder);
     configure(builder);
+    getActionManager().setup(builder);
```

Comment:
Makes sense now. I guess the reorder is the right way to solve. 

Clearly I didn't design this in a way that there is only one way of doing things. Will see if I can solve that somehow.